packages/radix-vue/src/Calendar/CalendarRoot.vue

```diff
@@ -81,6 +81,10 @@ interface BaseCalendarRootProps extends PrimitiveProps {
   isDateUnavailable?: Matcher
   /** The reading direction of the calendar when applicable. <br> If omitted, inherits globally from `ConfigProvider` or assumes LTR (left-to-right) reading mode. */
   dir?: Direction
+  /** A function that returns the next page of the calendar. It receives the current placeholder as an argument inside the component. */
+  nextPage?: (placeholder: DateValue) => DateValue
+  /** A function that returns the previous page of the calendar. It receives the current placeholder as an argument inside the component. */
+  prevPage?: (placeholder: DateValue) => DateValue
```

Comment:
we we have the default function to behave like `step:  'month'`? So dev dones't need to provide the props everytime

Reply:
Yes, the default is like `step: 'month'`.